image_metadata.json

```diff
@@ -0,0 +1 @@
vance": 51, "parentnames": ["Syria"]}, {"scheme": "http://cv.ap.org/id/", "code": "661e09b87d5b10048286c076b8e3055c", "name": "Syria", "creator": "Machine", "rels": ["direct"], "parentids": ["661850e07d5b100481f6c076b8e3055c"], "locationtype": {"code": "01f56e0e654841eca2e69bf2cbcc0526", "name": "Nation"}, "geometry_geojson": {"type": "Point", "coordinates": [38, 35]}, "relevance": 51, "parentnames": ["Middle East"]}], "description_caption": "Opposition forces take control of areas outside Aleppo, Syria, Friday, Nov. 29, 2024. (AP Photo/Ghaith Alsayed)", "renditions": {"main": {"title": "Full Resolution (JPG)", "rel": "Main", "format": "JPEG Baseline", "type": "picture", "contentid": "2e7c74034bae4a5fbb949539236ab9a6", "digest": "299d0266aef9d7ff9353c073c7f2aaa3", "href": "https://api.ap.org/media/v/content/fde0491a736b41caa5b632888a386c30.0/download?role=main&qt=tn1W8phJ9eF&cid=2e7c74034bae4a5fbb949539236ab9a6&ai=9c07da6f83036f34d4b18a479de9d085", "orientation": "Horizontal", "sizeinbytes": 4317478, "width": 5621, "height": 3748, "originalfilename": "Syria_Opposition_81304.jpg", "fileextension": "jpg", "mimetype": "image/jpeg", "pricetag": "Unlimited"}, "preview": {"title": "Preview (JPG)", "rel": "Preview", "format": "JPEG Baseline", "type": "picture", "contentid": "7da1749499454e4c955268f70e538325", "digest": "f9181bbed2c1409e3f392bccbab6d7da", "href": "https://mapi.associatedpress.com/v2/items/fde0491a736b41caa5b632888a386c30.0/preview/preview.jpg?nfe=true&s=512&wm=false&app=MPK&tag=iid~fde0491a736b41caa5b632888a386c30!rsn~0!cid~7da1749499454e4c955268f70e538325!orgId~124748!qt~tn1W8phJ9eF!orgNm~Five%20University%20Consortium!role~Preview!mt~photo!fmt~JPEG%20Baseline", "orientation": "Horizontal", "sizeinbytes": 98862, "width": 512, "height": 341, "originalfilename": "Syria_Opposition_81304.jpg", "fileextension": "jpg", "mimetype": "image/jpeg"}, "thumbnail": {"title": "Thumbnail (JPG)", "rel": "Thumbnail", "format": "JPEG Baseline", "type": "picture", "contentid": "8756f94ad2bc48498f174c4471dbe523", "digest": "5b2278ad0d94a6c3f02465e375814600", "href": "https://mapi.associatedpress.com/v2/items/fde0491a736b41caa5b632888a386c30.0/thumbnail/thumbnail.jpg?nfe=true&wm=false&app=MPK&tag=iid~fde0491a736b41caa5b632888a386c30!rsn~0!cid~8756f94ad2bc48498f174c4471dbe523!orgId~124748!qt~tn1W8phJ9eF!orgNm~Five%20University%20Consortium!role~Thumbnail!mt~photo!fmt~JPEG%20Baseline", "orientation": "Horizontal", "sizeinbytes": 38802, "width": 125, "height": 83, "originalfilename": "Syria_Opposition_81304.jpg", "fileextension": "jpg", "mimetype": "image/jpeg"}, "caption_nitf": {"title": "NITF Caption Download", "rel": "Caption", "format": "IIM", "type": "text", "mimetype": "text/xml", "fileextension": "xml", "words": 16, "contentid": "fde0491a736b41caa5b632888a386c30", "href": "https://api.ap.org/media/v/content/fde0491a736b41caa5b632888a386c30.0/download?qt=tn1W8phJ9eF&type=text&format=nitf&text=caption&et=0a1aza3c0&ai=9c07da6f83036f34d4b18a479de9d085"}}}}}
```

Comment:
I don't think we want the JSON files to be part of the commit, since they were more for illustration of what the data looks like. Could you remove them from the PR?